### ISSUE TITLE
Distinguish group vs individual mention vs unread

### ISSUE THREAD

**Issue by tobiasKaminsky** [nextcloud/spreed#5403]
Having lots of conversations, where many unread counters are blue does not help to find the most urgent ones.

Suggestion:
- grey: unread, but not mentioned
- blue border: group mentioned
- entire blue: individual mentioned

**Comment by nickvergessen** [comment]
The UI does not allow for border only atm and we also are not able to distinguish between mentions. It would require database and api modifications along the way and this is too much effort for a too little thing atm.

**Comment by schiessle** [comment]
I agree with the general problem described by Tobias. I have a hard time as well, keeping an overview.

Maybe an improved sorting could help already.

* Show rooms with unread messages in 1:1 chats always at the top (also above the favorites)
* Sort favorites so that rooms with unread messages are at the top of the favorites, everything else alphabetic
* All other rooms: sort them below the favorites again in a way that all unread are at the top of this group, everything else alphabetic

Don't have to scroll through the whole list to make sure that I don't miss any unread message would be already a great improvement, imho.

**Comment by nickvergessen** [comment]
> Don't have to scroll through the whole list to make sure that I don't miss any unread message would be already a great improvement, imho.

That's a completely different topic from my pov.
